src/figma/tokens.json

```diff
@@ -329,190 +329,190 @@
           "type": "color"
         }
       },
       "secondary": {
         "default": {
           "value": "#F66A0A",
-          "description": "(orange500: #F66A0A) secondary.default should be used for any secondary actions. It should not be used for any negative connotations such as warnings or errors as it is quite closely tied to the MetaMask Fox",
+          "description": "(orange500: #F66A0A) [Deprecated] secondary.default should be used for any secondary actions. It should not be used for any negative connotations such as warnings or errors as it is quite closely tied to the MetaMask Fox",
           "type": "color"
         },
         "alternative": {
           "value": "#C65507",
-          "description": "(orange600: #C65507) secondary.alternative should be used as an alternative to secondary.default for things such as hover states",
+          "description": "(orange600: #C65507) [Deprecated] secondary.alternative should be used as an alternative to secondary.default for things such as hover states",
           "type": "color"
         },
         "muted": {
           "value": "#F66A0A19",
-          "description": "(orange500: #F66A0A 10% opacity) secondary.muted is a very low contrasting secondary variant for things such as alert backgrounds. secondary.muted and secondary.inverse should not be used together in a foreground and background combination",
+          "description": "(orange500: #F66A0A 10% opacity) [Deprecated] secondary.muted is a very low contrasting secondary variant for things such as alert backgrounds. secondary.muted and secondary.inverse should not be used together in a foreground and background combination",
           "type": "color"
         },
         "inverse": {
           "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) secondary.inverse should be used only as the foreground element on top of secondary.default and secondary.alternative. It is intended to be the most contrasting color to secondary.default. It should meet all AA and AAA accessibility standards such as the text or icon of a secondary button.",
+          "description": "(white010: #FCFCFC) [Deprecated] secondary.inverse should be used only as the foreground element on top of secondary.default and secondary.alternative. It is intended to be the most contrasting color to secondary.default. It should meet all AA and AAA accessibility standards such as the text or icon of a secondary button.",
           "type": "color"
         },
         "disabled": {
           "value": "#F66A0A80",
-          "description": "(orange500: #F66A0A 50% opacity) secondary.disabled should be used for all disabled secondary action components",
+          "description": "(orange500: #F66A0A 50% opacity) [Deprecated] secondary.disabled should be used for all disabled secondary action components",
           "type": "color"
         }
       },
```

Comment:
What are we suggesting we update the current secondary color use case to?

<img width="359" alt="Screen Shot 2022-04-08 at 1 16 31 PM" src="https://user-images.githubusercontent.com/8112138/162525501-004e6017-c793-422b-9a3f-05346f8caefb.png">

![Screen Shot 2022-04-08 at 1 17 42 PM](https://user-images.githubusercontent.com/8112138/162525503-fc581322-0aca-45f6-91be-76891bc956de.png)

![Screen Shot 2022-04-08 at 1 17 50 PM](https://user-images.githubusercontent.com/8112138/162525506-aee21d84-f835-42c9-818d-801f4046c87b.png)

<img width="341" alt="Screen Shot 2022-04-08 at 1 19 12 PM" src="https://user-images.githubusercontent.com/8112138/162525507-6d8bbe7b-aebb-439c-bdc8-5265e37bad14.png">

Reply:
Resolved in internal chat.